plugins/http_plugin/http_plugin.cpp

```diff
@@ -13,26 +11,20 @@
 #include <boost/asio.hpp>
 #include <boost/optional.hpp>
 
-#include <websocketpp/config/asio_client.hpp>
-#include <websocketpp/config/asio.hpp>
-#include <websocketpp/server.hpp>
-#include <websocketpp/config/asio_client.hpp>
-#include <websocketpp/client.hpp>
-#include <websocketpp/logger/stub.hpp>
-
 #include <thread>
 #include <memory>
 #include <regex>
 
+#include "common.hpp"
+#include "beast_http_listener.hpp"
```

Comment:
Follow convention of using full relative path.

```
#include <eosio/http_plugin/common.hpp>
#include <eosio/http_plugin/beast_http_listener.hpp>
```
Which, of course, means moving those to the `include` directory.